Lib/contextlib.py

```diff
@@ -123,6 +128,7 @@ def __exit__(self, type, value, traceback):
                 #
                 if sys.exc_info()[1] is not value:
```

Comment:
@ncoghlan, is it worth to negate the condition and swap `raise` and `return` for uniformity with the above code? Shouldn't the function return `False` instead of `None` for uniformity?